application/F3DStarter.cxx

```diff
@@ -992,6 +996,30 @@ int F3DStarter::Start(int argc, char** argv)
       }
     }
 
+    // Process Command Script file
+    const std::string& commandScriptFile = this->Internals->AppOptions.CommandScriptFile;
+    if (!commandScriptFile.empty())
+    {
+      std::ifstream scriptFile(commandScriptFile);
+      if (scriptFile.is_open())
+      {
+        std::string command;
+        while (std::getline(scriptFile, command))
+        {
+          if (!command.empty())
+          {
+            interactor.triggerCommand(command);
+          }
+        }
+        scriptFile.close();
+      }
+      else
+      {
+        f3d::log::error("Unable to open command script file");
+        return EXIT_FAILURE;
```

Comment:
please add a test for this, you just need to point to a script that does not exist and look for the string in the output.